bugs/PENDING_SUBMIT/0397-BAD-IMAGE-S20PLUS-41991e5f-no-opt-test-SM-G986B-R3CNA03E21W/reduced_manual/variant/shader.frag

```diff
@@ -0,0 +1,122 @@
+#version 320 es
+
+precision highp int;
+
+precision highp float;
+
+// Contents of injectionSwitch: [0.0, 1.0]
+layout(set = 0, binding = 0) uniform buf0 {
+ highp vec2 injectionSwitch;
+};
+precision highp float;
+
+precision highp int;
+
+struct struct_base {
+ int data;
+ int leftIndex;
+ int rightIndex;
+} ;
+
+struct_base struct_a[10] = struct_base[10](
+                            struct_base(1, - 1, 1),
+                            struct_base(1, 1, 1),
+                            struct_base(1, 1, 1),
+                            struct_base(1, 1, 1),
+                            struct_base(1, 1, 1),
+                            struct_base(1, 1, 1),
+                            struct_base(1, 1, 1),
+                            struct_base(1, 1, 1),
+                            struct_base(1, 1, 1),
+                            struct_base(1, 1, 1)
+                            );
+
+void func_a(inout struct_base struct_a, int input_a)
+{
+ struct_a.data = 1;
+}
+void func_b(int input_b, int input_a)
+{
+ int var_a = 0;
+ int var_b = 0;
+ // Always true. Iterates 6 times,
+ // but returns nothing on first iteration.
```

Comment:
I don't think we can say that it iterates 6 times and/but it returns on the first iteration. It is one or the other. And I suppose we don't need to say that it is always true because we can see that based on the literals. So just say something like:

// Returns on the first iteration.